src/themes/default/pages/Product.vue

```diff
@@ -133,9 +133,12 @@
               v-if="product.type_id !== 'grouped' && product.type_id !== 'bundle'"
             >
               <base-input-number
-                :name="$t('Quantity')"
+                :name="$t(`${!isSimpleOrConfigurable ? `Quantity` : !isProductLoading ? `Quantity (${quantity} available)` : 'Getting information about quantity...'}`)"
```

Comment:
Needs to be translated using `i18n.t`/`$t` helper

Reply:
.. and please change to `Checking stock availability...`

Reply:
also with this condition it should be computed property which just returns correct name